integration/integration_suite_test.go

```diff
@@ -78,13 +79,14 @@ var _ = BeforeSuite(func() {
 		Address:                    fmt.Sprintf("%s/cephlet-volume.sock", os.Getenv("PWD")),
 		PathSupportedVolumeClasses: volumeClassesFile.Name(),
 		Ceph: app.CephOptions{
-			ConnectTimeout:       10 * time.Second,
-			Monitors:             os.Getenv("CEPH_MONITORS"),
-			User:                 os.Getenv("CEPH_USERNAME"),
-			KeyringFile:          os.Getenv("CEPH_KEYRING_FILENAME"),
-			Pool:                 os.Getenv("CEPH_POOLNAME"),
-			Client:               os.Getenv("CEPH_CLIENTNAME"),
-			KeyEncryptionKeyPath: keyEncryptionKeyFile.Name(),
+			ConnectTimeout:         10 * time.Second,
+			Monitors:               os.Getenv("CEPH_MONITORS"),
+			User:                   os.Getenv("CEPH_USERNAME"),
+			KeyringFile:            os.Getenv("CEPH_KEYRING_FILENAME"),
+			Pool:                   os.Getenv("CEPH_POOLNAME"),
+			Client:                 os.Getenv("CEPH_CLIENTNAME"),
```

Comment:
As an idea: can we set those values as global variables on top as they are also used in the subsequent tests like create volume test? That way we ensure that we only use the `Getenv` only once upon suite creation.